src/algebra/pointwise.lean

```diff
@@ -6,7 +6,7 @@ Authors: Johan Commelin
 Pointwise addition and multiplication of sets
 -/
```

Comment:
While you're at it, could you add a global docstring to this file, giving in particular the main definitions of this file and explaining that one needs to register a local instance to use them?